### DISCUSSION TITLE
T-POT - no disk drive was detected

### DISCUSSION THREAD

**Discussion by kalpesh23121983** [telekom-security/tpotce#864] [Q&A] [answered]
#196 , no disk drive was detected, if you know the name of the driver needed by your disk drive, you can select it from the list.

This error I will  facing in VM Ware ESXI.

**Comment by t3chn0m4g3** [comment, marked as answer]
I recommend to use the Post Install method.

**Reply by kalpesh23121983** [reply]
you mean to ay first install Debian os and then install t-pot right?

**Reply by t3chn0m4g3** [reply]
Yes https://github.com/telekom-security/tpotce#postinstall

**Reply by kalpesh23121983** [reply]
Thanks, Mate, For your Support,  it is work For Me 👍
